File: tools/get_disk_info.py
"""Disk usage information tool."""

from tools import tool
from tools._env_helpers import fmt


@tool(
    name="get_disk_info",
    description=(
        "Return C: drive disk usage: total, used, free space and usage percentage."
    ),
    parameters={"type": "object", "properties": {}, "required": []},
)
def get_disk_info() -> str:
    return fmt("disk")

File: tools/get_disk_info.ts
/** Disk usage information tool. */

import { registerInfoTool } from "./lib/env_helpers.js";

registerInfoTool({
  name: "get_disk_info",
  description:
    "Return all logical drives disk usage: total, used, free space and usage percentage per drive.",
  category: "disk",
});
